core/tests/protractor_utils/workflow.js

```diff
@@ -136,6 +130,13 @@ var getExplorationPlaytesters = function() {
   return _getExplorationRoles('viewer');
 };
 
+var getNumberOfFeedbackMessages = function() {
```

Comment:
Sorry, I don't understand. I thought that a creator would go into the exploration editor, and then click on the feedback icon in the menu on the top right? The creator dashboard just gives a deep link into the exploration editor page.